sdo/org.eclipse.persistence.sdo/src/main/java/org/eclipse/persistence/sdo/helper/SDOHelperContext.java

```diff
@@ -93,22 +94,13 @@ public class SDOHelperContext implements HelperContext {
     private String identifier;
     private Map<String, Object> properties;
     private boolean isStrictTypeCheckingEnabled = PrivilegedAccessHelper.getSystemPropertyBoolean(
-            STRICT_TYPE_CHECKING_PROPERTY_NAME, true);
+            SDOSystemProperties.SDO_STRICT_TYPE_CHECKING_PROPERTY_NAME, true);
 
-    /**
-     * Property controls strictness of {@link Type#getInstanceClass()} type checking.
-     *
-     * <p>
-     * See {@link #isStrictTypeCheckingEnabled()} for more details.
-     * By this property, the initial value can be changed.
-     * Default value is <code>true</code>.
-     * </p>
-     */
-    public static final String STRICT_TYPE_CHECKING_PROPERTY_NAME = "eclipselink.sdo.strict.type.checking";
```

Comment:
customers may be using this, so deprecate the field first and update its value to point to the new constant

Reply:
OK changed according by comment.